src/pages/rooms/[id]/detail.tsx

```diff
@@ -0,0 +1,27 @@
+import { GetServerSideProps, NextPage } from 'next';
+
+import RoomDetail from '@src/components/roomDetail/RoomDetail';
+
+interface Props {
+  id: string;
+}
+
+const detail: NextPage<Props> = (props) => {
+  return <RoomDetail></RoomDetail>;
+};
+
+const getServerSideProps: GetServerSideProps<Props> = async (ctx) => {
+  const {
+    query: { id },
+  } = ctx;
+
+  return {
+    props: {
+      id: id as string,
```

Comment:
```suggestion
  return {
    props: {
      id: String(id),
```

또는 `id.toString()`으로 형변환 해주면 에러안날 것 같은데 시도해봐주실수 있나요??

Reply:
오 배열도 toString메서드가 있는지 처음 알았네요 ㅎㅎ.
아 그리고 형변환 되는것 같습니다!! 형변환으로 적용해보겠습니다